tutorials/04-development.md

```diff
@@ -0,0 +1,9 @@
+### How to develop Enketo Core
+
+1. install [node](http://nodejs.org/) version 8 until [this issue](https://github.com/enketo/enketo-transformer/issues/53) is closed, and [grunt-cli](http://gruntjs.com/getting-started)
+2. install dependencies with `npm install`
+3. build with `grunt`
+4. start built-in auto-reloading development server with `npm start`
+5. browse to [http://localhost:8005](http://localhost:8005/) and load an XForm url with the `xform` queryparameter or load a local from from the /tests/forms folder in this repo
+6. run tests with `npm test`
+7. adding the querystring `touch=true` and reducing the window size allows you to simulate mobile touchscreens
```

Comment:
### Release a new version

Documentation is auto-generated and is re-built for each new release. Do not commit updated documentation in non-release commits. The process to follow for each release that includes various helpful checks is:

[ explain commands to use ]

[Explain the commands to use that perform the checks]